civitai-downloader-v2/tests/unit/test_data_models.py

```
@@ -0,0 +1,474 @@
+                'verify_ssl': True,
+                'rate_limit_requests_per_minute': 60
+            }
+        }
+        
+        config = AppConfig(**valid_config)
+        assert config.api.api_key.get_secret_value() == 'test_key_123'
+        assert config.api.timeout == 30
+        assert config.api.verify_ssl == True
+        
+        # Test validation constraints  
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)